lib/wit/main.py

```diff
@@ -227,8 +227,10 @@ def add_dep(ws, args) -> None:
     req_dep = dependency_from_tag(ws.root, args.pkg)
 
     cwd = Path(os.getcwd()).resolve()
-    cwd_dirname = cwd.relative_to(ws.root).parts[0]
+    if cwd == ws.root:
+        error("Cannot run add-dep from root of workspace.")
```

Comment:
I'm happy with this as is, but do you think it might be worth also suggesting `-C`?

Reply:
👍 on providing instructions on how to resolve the issue. Should we also add a description of _why_ running `add-dep` from the workspace root is an error? I think the error messages are an opportunity to teach users more about correct usage of the tool as well as bits and pieces of the mental model.

Reply:
Yeah agreed, I'll update the error and then merge.